etc/service-bus.api.ts

```diff
@@ -0,0 +1,411 @@
+  receiveDeferredMessages(sequenceNumbers: Long[]): Promise<ServiceBusMessage[]>;
+  receiveMode: ReceiveMode;
+  renewLock(): Promise<Date>;
+  sessionId?: string;
+  sessionLockedUntilUtc?: Date;
+  setState(state: any): Promise<void>;
+  // WARNING: The type "DispositionType" needs to be exported by the package (e.g. added to index.ts)
+  // WARNING: The type "DispositionOptions" needs to be exported by the package (e.g. added to index.ts)
+  settleMessage(message: ServiceBusMessage, operation: DispositionType, options?: DispositionOptions): Promise<any>;
+}
+
+// @public
+class MessagingError extends Error {
+  constructor(message: string);
+  condition?: string;
+  info?: any;
+  name: string;
+  retryable: boolean;
+  translated: boolean;
+}
+
+// @public
+class Namespace {
+  constructor(config: ConnectionConfig, options?: NamespaceOptions);
+  close(): Promise<any>;
+  // WARNING: The type "NamespaceOptionsBase" needs to be exported by the package (e.g. added to index.ts)
+  static createFromAadTokenCredentials(host: string, credentials: ApplicationTokenCredentials | UserTokenCredentials | DeviceTokenCredentials | MSITokenCredentials, options?: NamespaceOptions): Namespace;
+  static createFromConnectionString(connectionString: string, options?: NamespaceOptions): Namespace;
+  // WARNING: The type "NamespaceOptionsBase" needs to be exported by the package (e.g. added to index.ts)
+  // WARNING: The type "NamespaceOptionsBase" needs to be exported by the package (e.g. added to index.ts)
+  static createFromTokenProvider(host: string, tokenProvider: TokenProvider, options?: NamespaceOptionsBase): Namespace;
+  createQueueClient(queueName: string, options?: QueueClientOptions): QueueClient;
+  createSubscriptionClient(topicName: string, subscriptionName: string, options?: SubscriptionClientOptions): SubscriptionClient;
+  createTopicClient(topicName: string): TopicClient;
+  // (undocumented)
+  static getDeadLetterQueuePathForQueue(queueName: string): string;
+  name: string;
+}
+
+// @public
+interface NamespaceOptions extends NamespaceOptionsBase {
+  tokenProvider?: TokenProvider;
+}
+
+// @public
+export function parseConnectionString<T>(connectionString: string): ParsedOutput<T>;
+
+// @public (undocumented)
+class QueueClient extends Client {
+  // WARNING: The type "ConnectionContext" needs to be exported by the package (e.g. added to index.ts)
+  constructor(name: string, context: ConnectionContext, options?: QueueClientOptions);
+  // WARNING: The type "AcceptSessionOptions" needs to be exported by the package (e.g. added to index.ts)
+  // (undocumented)
+  acceptSession(options?: AcceptSessionOptions): Promise<MessageSession>;
+  cancelScheduledMessage(sequenceNumber: Long): Promise<void>;
+  cancelScheduledMessages(sequenceNumbers: Long[]): Promise<void>;
+  close(): Promise<any>;
+  // WARNING: The type "ListSessionsResponse" needs to be exported by the package (e.g. added to index.ts)
```

Comment:
"ListSessionsResponse" should be exported.

Reply:
`listMessageSessions` has been updated to return an array of string corresponding to the session ids, so this N/A anymore.